legend-engine-xts-snowflakeApp/legend-engine-xt-snowflakeApp-api/src/main/java/org/finos/legend/engine/language/snowflakeApp/deployment/SnowflakeAppArtifactGenerationExtension.java

```diff
@@ -21,12 +21,14 @@
 import org.finos.legend.engine.protocol.pure.v1.model.context.PureModelContextData;
 import org.finos.legend.pure.generated.Root_meta_external_function_activator_snowflakeApp_SnowflakeApp;
 import org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement;
-
+import org.slf4j.Logger;
+import java.util.Collections;
 import java.util.List;
 
 public class SnowflakeAppArtifactGenerationExtension implements ArtifactGenerationExtension
 {
     private static final String ROOT_PATH = "Snowflake App ";
+    private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger(SnowflakeAppArtifactGenerationExtension.class);
```

Comment:
@sprisha 
the root path for your artifact generation should be a valid file path. Therefore a space would not be valid, please fix.
private static final String ROOT_PATH = "Snowflake App ";

Reply:
got it, thanks